homeassistant/helpers/device_registry.py

```diff
@@ -384,6 +395,7 @@ class DeletedDeviceEntry:
     """Deleted Device Registry Entry."""
 
     config_entries: set[str] = attr.ib()
+    config_subentries: dict[str, set[str | None]] = attr.ib()
```

Comment:
I'd rename this to `config_entries_subentries` since it contains both and is used to know both things.